### PR TITLE
Feat(Ordering): Support RML OF 1604:14

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
### Added
-  Support for RML OF 1604:14

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] do submit an RML OF

### Expected test outcome
- [x] check that order was accepted
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by @northern-byte 
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Documented in AM
- [ ] Deploy this branch on cg-services-prod
- [ ] Inform to AG, AZ

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:
image
🆗 Order submitted ok

File: cg/apps/osticket.py [hunk 1]
```diff
@@ -37,6 +37,6 @@ def open_ticket(self, name: str, email: str, subject: str, message: str) -> Opti
             except ValueError:
                 LOG.error("Could not convert res %s to int", res.text)
                 return None
-        else:
```
Comment (northern-byte) [inline cg/apps/osticket.py:40]: Why was this removed?

**Review by northern-byte — APPROVED** [review]

File: cg/apps/osticket.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/apps/osticket.py:40]:
```
        if something:
            return
        else:
            raise
=
        if something:
            return
        raise
```

**Comment by patrikgrenfeldt** [comment]
Deployed to production: 
image

@annagellerbring @AnnaZetterlund
